mockredis/client.py

```diff
@@ -221,7 +222,9 @@ def do_expire(self):
         """
         Expire objects assuming now == time
         """
-        for key, value in self.timeouts.items():
+        # Deep copy to avoid RuntimeError: dictionary changed size during iteration
+        _timeouts = deepcopy(self.timeouts)
+        for key, value in _timeouts.items():
```

Comment:
I don't think we need a copy as we are using items() not iteritems() method. items is safe for deleting things

```
some_dict = {'one': 1, 'two': 2, 'three': 3, 'four': 4}
for k, v in some_dict.iteritems():
    if k == 'two':
        del some_dict[k]
```
## Response:
Traceback (most recent call last):
File "", line 1, in 
RuntimeError: dictionary changed size during iteration

```
some_dict = {'one': 1, 'two': 2, 'three': 3, 'four': 4}
for k, v in some_dict.items():
    if k == 'two':
        del some_dict[k]
```        

## Response:
some_dict
{'four': 4, 'three': 3, 'one': 1